app/templates/main/map.html

```diff
@@ -0,0 +1,122 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+<link rel="stylesheet" href="static/styles/map.css" type="text/css" />
```

Comment:
If you make `map.css` as a sass file and then move it to assets you won't have to import it here